Fenner_William-DFBT/js/index.js

```diff
@@ -21,19 +26,21 @@ function form_builder(attach, ...form_elements){
             form.innerHTML += "<"+form_elements[i][0]+" type='"+form_elements[i][1]+"'>"+form_elements[i][2]+"</ "+form_elements[i][0]+">";
         }
         else if(form_elements[i][0] == "select"){
-            form.innerHTML += "<"+form_elements[i][0]+">";
+            form.innerHTML += "<select>";
 
             for(var j=1; j<form_elements[i].length; j++){
                 form.innerHTML += "<option value='"+form_elements[i][j]+"'>"+form_elements[i][j]+"</option>";
             }
 
-            form.innerHTML += "</ "+form_elements[i][0]+">";
+            form.innerHTML += "</select>";
         }
 
     }
 
     if(attach == ""){
```

Comment:
The append to DOM approach is very cool, but project requires returning HTML string.